client/src/main/java/org/apache/celeborn/client/read/CelebornInputStream.java

```diff
@@ -715,6 +735,50 @@ public synchronized void close() {
       }
     }
 
+    void validateIntegrity() {
+      if (integrityChecked) {
```

Comment:
```suggestion
      if (integrityChecked || !shuffleIntegrityCheckEnabled ) {
        return;
```
There could be a lot of logs here because there could be a lot of InputStreams